evap/evaluation/models.py

```diff
@@ -444,7 +444,7 @@ class State:
     )
 
     class TextAnswerReviewState(Enum):
-        do_not_call_in_templates = True  # pylint: disable=invalid-name
+        do_not_call_in_templates = True
```

Comment:
Was the "disable" statement effectless? I would have expected pylint to fail on lines with a disable comment where the listed check does not actually trigger, so this should have made CI fail.

Reply:
well, ruff _should_ have catched this...

Reply:
Ah, so the pylint considers this a constant, which would have to be UPPER_CASE in pylint's interpretation of pep8 (since it's a [constant](https://peps.python.org/pep-0008/#constants)). Pylint probably would warn about a useless suppression. You removed the suppression because `invalid-name` is now disabled globally for pylint since ruff handles that part.

Quickly searching through the ruff rules, I couldn't find any that catches this. This implies to me that any name would be allowed here? I think pylint's interpretation of pep8 makes sense, and this should be a UPPER_CASE variable (if we could choose its name freely).

Reply:
And if I recall correctly, we weren't really happy with other parts of ruff variable name validation, right? So if we keep pylint anyways, we could also leave the naming stuff to pylint for now